Solved #14 - loading center issue

ankitgauri26 opened the pull request:
Corrected the CSS for loading.

ankitgauri26 commented:
@MaathavanJkr - Solved the issue. Kindly review

MaathavanJkr commented:
Nice. But the Loading can be more centered in Mobile Versions. in mobile.html rank.html study.html. 

see
![வரைபடவி - Varaipadavi - Google Chrome 10_22_2020 2_18_53 AM (2)](https://user-images.githubusercontent.com/17196210/96785851-07a02c00-140d-11eb-966f-705f3dfda356.png)

ankitgauri26 commented:
@MaathavanJkr - Loading seems to be centered in the Screenshot shared by you. Do you want to move loading up in "mobile.html rank.html study.html.".
I have changed the CSS which will effect in all the files and hence loading is coming as centered.. Previously when you zoom out, loading text is not aligned with loading icon. I have corrected that.
If still you want me to correct anything, request you to please elaborate more as loading seems centered for me in the screen shot above. What exactly needs to be done according to your screen shot

Matrix278 commented:
I think he wanted to have some space on mobile, as now it looks not kinda in center of mobile phone, like here https://github.com/kanimaitech/Varaipadavi/pull/18#issuecomment-713754358, I think he wants that padding space with that icon

ankitgauri26 commented:
@Matrix278 @MaathavanJkr - What I understand is loading icon and loading text should be in the center for all elements i.e. in the above screen it is in center from 3 sides (left, right and top) but not from bottom as it is very near to show button. Please let me know if my understanding is correct so that I can work on this?

MaathavanJkr commented:
See the dialog box. The loading icon is not centered vertically. The above
gap is greater than the below gap.

Maathavan

On Thu, 22 Oct 2020, 14:48 ankitgauri26, <notifications@github.com> wrote:

> @Matrix278 <https://github.com/Matrix278> @MaathavanJkr
> <https://github.com/MaathavanJkr> - What I understand is loading icon and
> loading text should be in the center for all elements i.e. in the above
> screen it is in center from 3 sides (left, right and top) but not from
> bottom as it is very near to show button. Please let me know if my
> understanding is correct so that I can work on this?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kanimaitech/Varaipadavi/pull/19#issuecomment-714355438>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEDGJMTIZFRZHIKO7I3WWZ3SL72E7ANCNFSM4S2EZCYQ>
> .
>

ankitgauri26 commented:
@MaathavanJkr - Thank you for the clarification. I'll work on it. Thanks

MaathavanJkr commented:
and if you can reduce the gap between the icon and loading text. Thank you

ankitgauri26 commented:
Corrected the CSS now. Kindly review
![image](https://user-images.githubusercontent.com/22481268/96888736-8fd11080-14a3-11eb-9a5a-84682ae4f8e3.png)

ankitgauri26 commented:
@MaathavanJkr - Kindly review the commits in the PR. Done 2 commit. 1 previous and 1 today.

ankitgauri26 commented:
@MaathavanJkr - Also reduced the gap between the icon and loading text. Thank you

MaathavanJkr commented on css/common.css:
```diff
@@ -422,4 +423,18 @@ td.rank {
+}
+
+
+#quiz-loader, 
```
Instead of this we can just add text-center class to these elements?

ankitgauri26 replied:
Added "text-center" class to these elements. Thank you for the review

MaathavanJkr commented on rank.html:
```diff
@@ -49,7 +49,7 @@
             <div></div>
         </div>
-        <div class="text-center">Loading</div>
+        <div class="loading-text text-center">Loading</div>
```
then we wont need this text-center class here ig

ankitgauri26 replied:
Removed text-center from loading as we have applied text-center in parent element. Thank you for the review

MaathavanJkr reviewed (changes requested):
Awesome work. :) Consider this too ;)

ankitgauri26 commented:
@MaathavanJkr - Done the changes as per review. Thank you again for the suggestions.

MaathavanJkr commented:
@ankitgauri26 Great Work. Thank you. :)
